### PR TITLE
doc: Add Zabbix ceph-mgr plugin to PendingReleaseNotes

### PR DESCRIPTION
**Pull request by wido**
Add Zabbix ceph-mgr module to release notes as it has been merged earlier.

Signed-off-by: Wido den Hollander <wido@42on.com>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -210,3 +210,7 @@
 * The "ceph -w" output no longer contains audit log entries by default.
   Add a "--watch-channel=audit" or "--watch-channel=*" to see them.
 
+* ceph-mgr now has a Zabbix module. Using zabbix_sender it sends trapper
```
Comment (tchaikov) [inline PendingReleaseNotes]: s/module/plugin/ for better consistency.

Comment (wido) [inline PendingReleaseNotes]: Thanks! Pushed a new commit with the fix in it

**Comment by tchaikov** [comment]
@wido needs rebase.

**Comment by wido** [comment]
Done @tchaikov

**Comment by tchaikov** [comment]
jenkins test doc

**Pull request merged**